public/config.default/theme/picoman.php

```diff
@@ -0,0 +1,230 @@
+<script>
+/*!
+ * PicoMan v1.0 (https://mamans86.blogspot.com)
+ * email: mamansulaeman86@gmail.com telegram: @mamans86
+ * feel free to modify but please keep link to my blog always visible and reachable, and also please don't remove this comment block
+ * contact me if you want to make donation
+ * TCExam Mobile First Design Theme based on Pico.css (https://picocss.com), built with love by Maman Sulaeman - @mamans86
+ * eye & flag icon generated by icomoon.io
+ */
```

Comment:
It is OK to state the author but not additional licensing conditions.
Please refer to the TCExam license + https://cla-assistant.io/tecnickcom/tcexam?pullRequest=350